### PR TITLE
Add previews of game assets

### PR DESCRIPTION
**Pull request by maror757**
Fixed issue #3,

Ignore my previous PR for this issue as I accidentaly extended the game asset preview branch on top of another PR which was directed towards issue #13.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,59 @@
...
+<div>
+  <img src="images/anim-sheet.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/apple.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/birch_planks.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/clay.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/cobblestone.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/gold_block.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/health.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/inventory.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/state-transition.png?raw=true" width=50 height=50 hspace=3>
```
Comment (silwalanish) [inline README.md]: Can you remove this?

File: README.md [hunk 2]
```diff
@@ -14,3 +14,59 @@
 1. Ground Textures: https://opengameart.org/content/2d-minecraft-sprite-pack
 2. Steve Textures: https://opengameart.org/content/2d-minecraft-steve
 3. Zombie Sprite: https://opengameart.org/content/zombie-character-sprite
+
+Minecraft
+<div>
+  <img src="images/minecraft/Spritesheet.PNG?raw=true" width=50 height=50 hspace=3>
```
Comment (silwalanish) [inline README.md]: Let's not show spritesheet as a 50x50. Let's show them in full size.

File: README.md [hunk 3]
```diff
@@ -14,3 +14,59 @@
...
+  <img src="images/skies/ora.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/skies/sen.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/skies/sno.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/skies/son.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/skies/twi.png?raw=true" width=50 height=50 hspace=3>
+</div>
+
+General
+<div>
+  <img src="images/anim-sheet.png?raw=true" width=50 height=50 hspace=3>
```
Comment (silwalanish) [inline README.md]: .

File: README.md [hunk 4]
```diff
@@ -14,3 +14,59 @@
...
+  <img src="images/apple.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/birch_planks.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/clay.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/cobblestone.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/gold_block.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/health.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/inventory.png?raw=true" width=50 height=50 hspace=3>
```
Comment (silwalanish) [inline README.md]: .

File: README.md [hunk 5]
```diff
@@ -14,3 +14,59 @@
...
+  <img src="images/state-transition.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/zombie_typea_attack_spritesheet.png?raw=true" width=50 height=50 hspace=3>
+  <img src="images/zombie_typea_walk_spritesheet.png?raw=true" width=50 height=50 hspace=3>
```
Comment (silwalanish) [inline README.md]: .

File: README.md (hunk 2 above)
Comment (maror757) [reply README.md]: I fixed this for the spritesheets in the "general" section. I removed the one in "Minecraft" as it just showed the same assets that comes right after.

File: README.md (hunk 1 above)
Comment (maror757) [reply README.md]: It has been removed.

**Review by silwalanish — APPROVED** [review]

**Comment by silwalanish** [comment]
Thanks for the contribution :heart:. But closing this PR :cry: with #17 even though that came late. Not that this didn't fulfill the requirement but #17 went beyond my expectation. Hope to get more contribution in the future too. :100: :tada: No hard feelings :1st_place_medal:

**Pull request closed without merging**